moveit_core/package.xml

```diff
@@ -25,7 +25,7 @@
   <build_depend>eigen_conversions</build_depend>
   <build_depend>eigen_stl_containers</build_depend>
   <build_depend>libfcl-dev</build_depend>
-  <build_depend version_gte="0.3.4">geometric_shapes</build_depend>
+  <build_depend version_gte="0.5.1">geometric_shapes</build_depend>
```

Comment:
instead of enforcing the version with some static_assert during compilation, we should simply specify the correct version here.